widget/entry_test.go

```diff
@@ -448,8 +448,8 @@ func TestEntry_TappedSecondary(t *testing.T) {
 	pos := fyne.CurrentApp().Driver().AbsolutePositionForObject(over)
 
 	cont := over.(*PopUp).Content
-	assert.Equal(t, pos.X+theme.Padding()+tapPos.X, cont.Position().X)
```

Comment:
This doesn't smell right. Are entry popups now not padded, or is the position handling changed?

Reply:
The padding is now in the menuItemWidget and therefore the pop-up must not add additional padding.

Reply:
I don't follow - this code is asking for the X position of the top overlay. Why does where the padding is change that?

Reply:
Oh, I think I see - you are comparing to the content position, and that is now different?

I guess I missed the implicit design change "popup menus no longer have padding".

Reply:
Its explicit in the commit comment:
“… menu pop-up is without padding”

Reply:
Haha, yes I guess. Perhaps changes in design could be called out with an image?
I know it's not a big deal just for some reason it hadn't sunk in.